### PR TITLE
Test "populate" commands

### LINKED ISSUE #139: populate command needs unit-testing
continuation from https://github.com/dandi/dandisets/pull/137#issuecomment-1070943139

### PR DESCRIPTION
**Pull request by jwodder**
Closes #139.

### CODE REVIEW

File: tools/backups2datalad-update-cron [hunk 1]
```diff
@@ -30,17 +28,13 @@ set -x
 eval python -m tools.backups2datalad \
    -l WARNING \
    -J 5 \
-   --target "$ds" \
+   --backup-root "$ds" \
```
Comment (yarikoptic) [inline tools/backups2datalad-update-cron]: `ds` is /mnt/backup/dandi/dandisets/ I believe, but `path` within `tools/backups2datalad.cfg.yaml` below has  `path: dandisets` and `path: dandizarrs` assuming it is ` /mnt/backup/dandi/`, so either config or these cron scripts should be adjusted (if you while at it create some `backups2datalad-cron-common` to be sourced by all others to set variables -- would be appreciated)

Comment (jwodder) [reply tools/backups2datalad-update-cron]: Fixed & done.

**Comment by yarikoptic** [comment]
Thanks! Will merge/do test runs while supervising tomorrow!

**Comment by yarikoptic** [comment]
Away from laptop ATM at a meeting, didn't check, but I believe cron job is disabled ATM, so should be safe - let's proceed. Thank you @jwodder

**Pull request merged**